src/components/ExecuteButton.js

```diff
@@ -111,7 +111,10 @@ export class ExecuteButton extends React.Component {
       } else {
         document.removeEventListener('mouseup', onMouseUp);
         onMouseUp = null;
-        this.setState({ optionsOpen: false });
+        // don't run setState if we click on an <li> menu item
+        if (upEvent.target.parentNode.parentNode !== downTarget.parentNode) {
```

Comment:
A small nit, but `upEvent.target.parentNode.parentNode` could be named into something more readable